packages/components/src/ClearButton/ClearButton.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { ButtonHTMLAttributes } from "react"
 import classnames from "classnames"
-import { ClearIcon } from "~components/SVG/icons/ClearIcon"
+import { ClearIcon } from "~icons/ClearIcon"
```

Comment:
So is our convention to use `~components/Icons` or `~icons/*Icon`? I think it's better to just stick to one.

Reply:
Format still seems inconsistent across the files, which did we land on?